gspread_asyncio/__init__.py

```diff
@@ -880,11 +880,13 @@ async def reorder_worksheets(
         )
 
     @property
-    def sheet1(self) -> str:
+    async def sheet1(self) -> "AsyncioGspreadWorksheet":
         """:returns: Shortcut property for getting the first worksheet.
-        :rtype: str
+        :rtype: an instance of :class:`AsyncioGspreadWorksheet`.
         """
-        return self.ss.sheet1
+        ws = await self.agcm._call(lambda: self.ss.sheet1)
+        aws = self._wrap_ws(ws)
+        return aws
```

Comment:
could also simply do this
```suggestion
        return await self.get_worksheet_by_id(0)
```